packages/gatsby/src/internal-plugins/query-runner/file-parser.js

```diff
@@ -15,14 +14,6 @@ import type { DocumentNode, DefinitionNode } from "graphql"
 
 const readFileAsync = Bluebird.promisify(fs.readFile)
 
-function getAssignedIdenifier(path) {
```

Comment:
My VSCode tell me that is not used, but you can recheck ;) (I try to remove unused code when I see it)